test/common/alerts_exist.go

```diff
@@ -895,6 +901,18 @@ func managedApiAwsExpectedRules(installtionName string) []alertsTestRule {
 }
 
 func TestIntegreatlyAlertsExist(t TestingTB, ctx *TestingContext) {
+	prometheusPod := "prometheus-application-monitoring-0"
+	execToPodCommand := "curl localhost:9090/api/v1/rules"
+	testIntegreatlyAlertsExist(t, ctx, MonitoringOperatorNamespace, prometheusPod, execToPodCommand)
+}
+
+func TestIntegreatlyAlertsExistObservability(t TestingTB, ctx *TestingContext) {
+	prometheusPod := "prometheus-kafka-prometheus-0"
+	execToPodCommand := "wget -qO - localhost:9090/api/v1/rules"
```

Comment:
just out of curiosity - is there a reason to have `curl` and `wget` there? I mean why both and not any particular? :thinking:

Reply:
Hi Maksym. The reason is that prometheus-kafka-prometheus-0 pod in OO is missing curl